src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt

```diff
@@ -20,18 +20,30 @@ import com.xenomachina.argparser.ArgParser
 import com.xenomachina.argparser.default
 import com.xenomachina.argparser.mainBody
 import org.slf4j.LoggerFactory
+import java.io.File
 
 /**
  * Simple server that will wait for request and will send response back.
  * It uses CosmasGoogleCloudService or CosmasInMemoryService to store files
  * @author Aleksandr Fedotov (iisuslik43)
  */
 class CosmasServer(port: Int, val service: CosmasGrpc.CosmasImplBase) {
-    private val server: Server = ServerBuilder
+
+    constructor(port: Int, service: CosmasGrpc.CosmasImplBase,
+                certChain: File, privateKey: File) : this(port, service) {
+        this.server = ServerBuilder

```

Comment:
Add Preconditions calls checking that files exist?

Reply:
done